src/index.html

```diff
@@ -1,13 +1,50 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div 
+      class="card" 
+      data-qa="card"
+    >
+      <img 
+        class="card__image"
+        src="./images/imac.jpeg" 
+        alt="APPLE A1419 iMac 27&quot; Retina 5K Monoblock (MNED2UA/A)"
+      >
+      <h1 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h1>
+      <p class="card__code-text">Product code: 195434</p>
+      <div class="box">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <p class="color-text">Reviews: 5</p>
+      </div>
+
+      <div class="prises-container">
+        <h2 class="prise-text">Price:</h2>
+        <p class="prise">$2,199</p>
+      </div>  
+
+      <div class="button">
```

Comment:
```suggestion
      <div class="card__review">
        <div class="stars stars--4">
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
        </div>
        <p class="card__rewiew-text">Reviews: 5</p>
      </div>

      <div class="card__price">
        <h2 class="card__price-text">Price:</h2>
        <p class="card__price-amount">$2,199</p>
      </div>  

      <div class="card__button">
```
in this task there should be only card and stars BEM blocks